src/main/java/org/aksw/simba/squirrel/metadata/CrawlingActivity.java

```diff
@@ -139,17 +220,41 @@ public Worker getWorker() {
         return worker;
     }
 
-    public enum CrawlingURIState {SUCCESSFUL, UNKNOWN, FAILED;}
+    public enum CrawlingURIState {SUCCESSFUL, UNKNOWN, FAILED}
 
     public CrawlingURIState getState() {
         return state;
     }
 
-    public CrawleableUri getCrawleableUri() {
-        return uri;
+    public URI getUrl (String o)
+    {
+            try {
+                URL Domain = new URL("http://www.example.org/");
+                URL url = new URL(Domain + o);
```

Comment:
Why are you creating a URL object from a string when you are only using it as a string in the next step?